### ISSUE TITLE
Update status of Logs Bridge API/SDK

### ISSUE THREAD

**Issue by tigrannajaryan** [open-telemetry/opentelemetry.io#2639]
The status shows as "Experimental", needs to be changed to "Stable": https://opentelemetry.io/docs/reference/specification/logs/

**Comment by svrnm** [comment]
All updates to /docs/reference/specification happen when a new spec release is done, so this will update accordingly when a new release is done.

We had a similar situation the other day with the sunsetting of OpenCensus that something stable in the spec repo was still experimental in the version visible on the website. Just thinking out lout, but maybe we should add something to the spec on the website calling out the current version and letting people know that the most recent instance can be found on github. cc @cartermp @austinlparker @chalin

**Comment by chalin** [comment]
> maybe we should add something to the spec on the website calling out the current version

Yes, that is what I had in mind too, but was hoping to add that feature at the same time as other work on:

- #2387

Since #2387 hasn't gotten any traction from spec folks, I can work on separately addressing the spec-version issue (though it will require manually updating the spec version in such a case). Thoughts / comments or preferences?

**Comment by svrnm** [comment]
@chalin let's not address it separately. 

@tigrannajaryan any thoughts on #2387?

**Comment by chalin** [comment]
Can we close this issue given that the following has landed?

- #2666

**Comment by cartermp** [comment]
Yes, let's close this as it'll be addressed promptly with the spec update.
